packages/editor-ui/src/components/Node/NodeCreator/TriggerHelperPanel.vue

```diff
@@ -368,18 +370,26 @@ function onActionSelected(actionCreateElement: INodeCreateElement) {
 	setAddedNodeActionParameters(actionUpdateData, telemetry);
 }
 function addHttpNode() {
+	const app_identifier = state.activeNodeActions?.name;
```

Comment:
Why snake case?

Reply:
[Not added as part of this PR](https://github.com/n8n-io/n8n/commit/79fe57dad8093b27651ce82164d6e7a0f08f9e43#diff-9940be827221abeed395c8fd6da98f6f6ac693a32f8cd9f9422ad1d8fd9710f3R321).